### PR TITLE
Added Changes to CONTRIBUTE.md

### PR DESCRIPTION
**Pull request by praveen-221**
## Summary
Add how to pull latest changes to local branch to contributing.md, This closes the issue #22 and creating PR under hacktoberfest 2022

## Checklist
- [x] tested locally.
- [x] No error nor warning in the console.

### CODE REVIEW

File: CONTRIBUTE.md [hunk 1]
```diff
@@ -23,15 +23,55 @@ or navigate to the respective folder and `yarn add <package>`
...
+   git remote -v
+```
+
+**4.** Fetching from the remote repository will bring in its branches and their respective commits.
+
+```
+   git fetch upstream
+```
+
+**5.** Make sure that you're on your main branch.
```
Comment (cherylli) [inline CONTRIBUTE.md:46]: this also works if you want to pull latest changes to your current working branch which is not main, resolve conflicts if you have to

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -23,15 +23,55 @@ or navigate to the respective folder and `yarn add <package>`
...
+   git merge upstream/main
+```
+
+**7.** Or you can club step 4th and 6th together using:-
+
+```
+   git pull upstream main
+```
+
+
```
Comment (cherylli) [inline CONTRIBUTE.md:64]: maybe add a note saying they need to push the changes back to the their own forked repository to update it on github

**Review by cherylli — CHANGES_REQUESTED** [review]
thank you. it looks good but I would like to add some notes to it

I also think the upstream is automatically added when you fork and clone but good to check that it's already there!

File: CONTRIBUTE.md (hunk 2 above)
Comment (praveen-221) [reply CONTRIBUTE.md]: In order to get commits done by other contributors, I have made the quoted changes and opened a new PR #46 
Thank you

**Pull request closed without merging**